demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -240,13 +240,14 @@ def _is_pack_meta_file_structure_valid(self):
 
             # check created field in iso format
             created_field = metadata.get(PACK_METADATA_CREATED, '')
-            if not self.check_timestamp_format(created_field):
-                suggested_value = parser.parse(created_field).isoformat() + "Z"
-                if self._add_error(
-                        Errors.pack_timestamp_field_not_in_iso_format(PACK_METADATA_CREATED,
-                                                                      created_field, suggested_value),
-                        self.pack_meta_file):
-                    return False
+            if created_field:
```

Comment:
When will this condition be true?

Reply:
It will be true if the pack_metadata file includes `created` field.

We dont remove `created` field from old content, so we still need to have this check, but we first need to check that there is a `created` field in the pack_metadata, if so we want to check that it is in iso format.